replica/table.cc

```diff
@@ -2743,6 +2743,10 @@ future<bool> table::snapshot_exists(sstring tag) {
 future<std::unordered_map<sstring, table::snapshot_details>> table::get_snapshot_details() {
     return seastar::async([this] {
         std::unordered_map<sstring, snapshot_details> all_snapshots;
```

Comment:
> This is continuation of https://github.com/scylladb/scylladb/commit/24589cf00cf8f1fae0b19a2ac1bd7b637061301a and https://github.com/scylladb/scylladb/commit/a734fd5c9cf254fb63180482875808be2bd77dbc -- if table is not
based on local storage, getting snapshot details makes no sense.


Why not? It makes sense to snapshot on object storage too.

We could do it by using CopyObject, or by adding references in the system.sstables table to avoid garbage colllection, or a combination.

It's okay to say we don't support it now.

Reply:
Yes, I wasn't very correct here. It makes no sense with current implementation that scans datadir unconditionally, and s3-backed sstables don't have this directory at all